files/slack.rb

```diff
@@ -98,8 +102,11 @@ def message
     }
 
     compact_msg = {
-      "username" => "Sensu (#{Socket.gethostname.split('.')[0]})",
-      "text"     => description(maxlen=400)
+        "attachments" => [{
```

Comment:
the point of having a compact display is specifically to avoid the overhead of having attachments. will the expanded format not work for you here?

perhaps there is a middle ground?

Reply:
the full display is too big,  can clutter chat. 

this attachment is explicitly small height, not any taller than the non attachments version of the message.  it just gets a color bar in front of it. 

![image](https://user-images.githubusercontent.com/1090063/39944418-7c98c9bc-551b-11e8-9de5-a048f538e43a.png)

 is there another overhead of attachments we're not aware of?    we could rejigger this to be a third display option.    but it seemed beneficial to all who use compact?

Reply:
@fhats we anxiously await clarification of your concern so we can find that middle ground. :)  :P  :)

Reply:
sorry, it was the weekend :)

the format actually looks good to me; let's just be cautious about adding anything else that is likely to expand the messages.

Reply:
yeah, I agree with you on the compactness. :)